draft-ralston-mimi-terminology.md

```diff
@@ -146,7 +146,7 @@ what is needed to construct an MLS Group Context.
 identify them at least within the group. These are essentially what a user would call a
 "message", though specifically the unencrypted portion. When encrypted, they are called events.
 
-**Server**: Synonymous with an MLS Delivery Service (DS). Responsible for routing events
+**Server**: Responsible for routing events
```

Comment:
the diff is just slightly too narrow for me to suggest a diff, but I think it'd be good to clarify the DS somewhere in this document. We can also drop the p2p clause while we're here, given MIMI has largely agreed that it's not a thing the WG wants to pursue.

Suggested definition would be:

> **Server**: Responsible for routing events to other servers and local clients. The collection 
> of servers and clients in a chat form the MLS Delivery Service (DS).

Reply:
I don't object to your rewording, but in general I find the DS term to be so unhelpful and counterintuitive that I prefer to only mention it only briefly in the transport protocol and ignore everywhere else.

Reply:
Fair. We should move the DS definition to the working group's transport document when it has one.